include/seqan3/alignment/configuration/align_config_method.hpp

```diff
@@ -0,0 +1,77 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides global and local alignment configurations.
+ * \author Joshua Kim <joshua.kim AT fu-berlin.de>
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \author Jörg Winkler <j.winkler AT fu-berlin.de>
+ * \author Wiep van der Toorn <w.vandertoorn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alignment/configuration/detail.hpp>
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+#include <seqan3/core/detail/empty_type.hpp>
+
+
+namespace seqan3::detail
+{
+    //!\brief A strong type to select the global alignment method.
+    //!\ingroup alignment_configuration
+    struct method_global_tag : public pipeable_config_element<method_global_tag, empty_type>
```

Comment:
```suggestion
    struct method_global_tag : public pipeable_config_element<method_global_tag>
```
I thinks this is already possible. can you check ?

Reply:
Yes, it does, nice